modules/ROOT/pages/using-the-api-console.adoc

```diff
@@ -0,0 +1,34 @@
+= Using the API Console 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The API Console is an interactive component that shows information about REST APIs and their methods, and enables you to test the methods by sending requests. The API Console is located in the API Details page. Endpoints are displayed in alphabetical order.
+
+The API Console displays the API specifications and documentation. An API’s specifications and documentation information are critical to the onboarding process and adoption. Developers need to know how to use and integrate APIs. 
+
+The Specification pane displays the navigation of the HTTP functions in the REST specification for a REST API. You can understand the format and structure of an output by reviewing the resources and methods. 
+
+The Documentation pane displays interactive documentation about an API. API documentation includes detailed descriptions of each API version. This documentation is updated with the API release cycle, usually with close involvement from the development team. You can also review the request information and responses.
+
+The Try it pane enables you to test REST APIs using the mocking service or an API instance URL. The mocking service lets you test endpoints defined in the API specification without an actual server to handle the requests and responses. The mocking service enables you add simulated data to an API function and test the API with that data. All the responses are example data. Using an API instance enables you to interact with a server and get real data for real requests and responses.
```

Comment:
The mocking service enables you `to` add simulated data...

Reply:
Thanks for the thorough review!